doc/man7/EVP_SIGNATURE-ED25519.pod

```diff
@@ -76,12 +76,23 @@ EVP_PKEY_CTX_get_params().
 
 =over 4
 
+=item * "verify-digested" (B<OSSL_SIGNATURE_PARAM_EDDSA_VERIFY_DIGESTED> <integer>
+
+=back
```

Comment:
back seems in the wrong place here

Reply:
It's matching elsewhere in the file.  The points are indented, the descriptive text isn't.

Reply:
The item is in the wrong place however.